Kernel/API/Syscall.h

```diff
@@ -363,17 +363,8 @@ struct SC_create_thread_params {
     unsigned int reported_guard_page_size = 0; // The lie we tell callers
     unsigned int stack_size = 1 * MiB;         // Equal to Thread::default_userspace_stack_size
     void* stack_location;                      // nullptr means any, o.w. process virtual address
-#    if ARCH(X86_64)
-    FlatPtr rdx;
-    FlatPtr rdi;
-    FlatPtr rbx;
-    FlatPtr rsi;
-#    elif ARCH(AARCH64)
-    FlatPtr x1;
-    FlatPtr x2;
-    FlatPtr x3;
-    FlatPtr x4;
-#    endif
+    FlatPtr entry;
```

Comment:
…doesn't this change render the previous two patches moot? Maybe squash the first 3 commits together?

Reply:
Got out to #17205 – excuse me for the commotion! 🙇🏻‍♂️